mail-client/clawsker/clawsker-1.3.1.ebuild

```diff
@@ -0,0 +1,23 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Applet to edit Claws Mail's hidden preferences"
+HOMEPAGE="https://www.claws-mail.org/clawsker.php"
+SRC_URI="https://www.claws-mail.org/tools/${P}.tar.xz"
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="
+	dev-lang/perl
+	dev-perl/Gtk3
+	dev-perl/Locale-gettext
+	dev-perl/File-Which
```

Comment:
I didn't highlight this for a reason, because there actually seem to be older version in the tree :P 
but I'll add the restriction, LGTM otherwise. Thanks!

```
 * QA Notice: new icons were found installed but icon cache
 * has not been updated:
 *   /usr/share/icons/hicolor/48x48/apps/clawsker.png
 *   /usr/share/icons/hicolor/128x128/apps/clawsker.png
 *   /usr/share/icons/hicolor/64x64/apps/clawsker.png
 * Please make sure to call xdg_icon_cache_update()
 * in pkg_postinst() and pkg_postrm() phases of appropriate pkgs.
```
I'll fix that, too.

Reply:
Thanks, I will check the final file to see your fixes then !